### DISCUSSION TITLE
Idea: Add click to drag on History Events vertical bar.

### DISCUSSION THREAD

**Discussion by superdong69** [blakeblackshear/frigate#11726] [Ideas]
![image](https://github.com/blakeblackshear/frigate/assets/119988539/664b315a-ab87-4a2a-9c30-74c28836f3b4)

The event scrub bar on the right, you can drag the time portion, but what about being able to click and drag the timeline?
I was just scrubbing through a camera history looking for events, and at 5pm, to get back to start looking around 7am is a chore. Would be awesome to be able to grab the timeline like in an iphone, and sling it to what it going sailing backwards or forwards. I know this is poorly explained, but almost rubber bandy, where it has weight to it like the iphone does. Not a game breaker change I know, but certainly would be a nice feature to be able to quickly sling through the day and see the marks in the timeline where events were detected, then grab the time slider, and pinpoint what you want to watch.

**Comment by NickM-27** [comment]
> I was just scrubbing through a camera history looking for events, and at 5pm, to get back to start looking around 7am is a chore.

you can just use your scroll wheel to move the timeline and then click where you want the time to jump to

**Reply by superdong69** [reply]
LMAO. Wow. I feel REALLY stupid. It was a good idea though! You guys just made it easier. An idea though if you all decide on a mobile app! HAHA

**Reply by hawkeye217** [reply]
You can scroll the timeline in the same way on mobile :)

**Reply by superdong69** [reply]
Nevermind then! HAHA
